control_panel_api/management/commands/add_oidc_statement_to_users_roles.py

```diff
@@ -0,0 +1,65 @@
+import json
+import logging
+
+import boto3
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from control_panel_api.models import User
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Adds OIDC statement to all users roles trust policies"
+
+    def handle(self, *args, **options):
+        iam = boto3.resource('iam')
+        for user in User.objects.all():
+            # Read role trust policy
+            iam_role = iam.Role(user.iam_role_name)
+            assume_role_policy_document = iam_role.assume_role_policy_document
+
+            if not self._has_oidc_statement(assume_role_policy_document):
```

Comment:
Generally, it's clearer to test for truth (avoid `if not`). I would invert the logic here (and switch the if/else blocks).